app/javascript/src/Stats/lib/stats.jsx

```diff
@@ -51,7 +55,7 @@ export const Stats = function ({ChartManager, Chart, Sources}) {
       let statsState = new StatsState(store, Object.assign({}, DEFAULT_STATE, selectedState))
 
       metrics.then(list => {
-        let groupedMetrics = settings.hasGroupedMethods ? list.metrics : [list.metrics[0], ...list.methods, ...list.metrics.slice(1)]
+        let groupedMetrics = settings.hasGroupedMethods ? list.metrics : groupMetrics(list)
```

Comment:
here `const` too. I know previously it's been defined with a `let` but since we are changing it now we could use the oportunity